common/common.go

```diff
@@ -380,6 +380,15 @@ func writeTmpDockerfile(helper langs.LangHelper, dir string, ff *FuncFile) (stri
 		dfLines = append(dfLines, fmt.Sprintf("FROM %s", ri))
 		dfLines = append(dfLines, "WORKDIR /function")
 		dfLines = append(dfLines, helper.DockerfileCopyCmds()...)
+		
+		resources := ff.CopyResources
+		for _, resource := range resources {
+			if Exists(resource.From){
+				dfLines = append(dfLines, fmt.Sprintf("COPY %s %s", resource.From, resource.To))
+			}else{
```

Comment:
Can you runt he linter? There are some minor problems with the formatting such as with the spacing.